docs/_includes/head_custom.html

```diff
@@ -9,6 +9,5 @@
 <!-- Fonts -->
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Fira+Sans&display=swap" rel="stylesheet">
-<link href="https://fonts.googleapis.com/css2?family=Fira+Mono&display=swap" rel="stylesheet">
-<link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;900&display=swap" rel="stylesheet">
```

Comment:
@gongjason

Reply:
I thought it was a bit more readable and "technical docs" looking than Fira. I think it's the lack of curly tails on the letters...

![image](https://user-images.githubusercontent.com/52545545/172492476-121c9522-20ec-4339-831c-443619175429.png)
![image](https://user-images.githubusercontent.com/52545545/172492489-edf37dcd-8d00-4553-9943-1a89e6276d7a.png)

What do you think?

Reply:
I think it looks good too, but doesn't look like it [has a `monospace` variant](https://github.com/rsms/inter/issues/128). We could use it just for body text, but now we have 3 font families: `Open Sans`, `Fira`, and `Inter`.

If we're making the change here, should also make it in the product and website to be consistent. We originally went with Fira because that's the original font used in Firefox (has an open-source feeling to it). Another option we could switch to could be [`Ubuntu`](https://fonts.google.com/specimen/Ubuntu) which would be familiar to any Linux user.

I don't necessarily think the `Inter` font looks bad -- I guess my point is that consistency and availability of all font styles should maybe trump what subjectively looks good to our eyes

Reply:
👍 Ubuntu looks pretty good. I think we can make a separate issue/PR for this. I'll revert back to Fira for now.

It looks like the [Firefox/Mozilla](https://www.mozilla.org/en-US/) have moved to Inter too 😂